test/e2e/util/util.go

```diff
@@ -280,6 +285,42 @@ func WaitForClusterOperatorConditionReason(cs *framework.ClientSet, interval, du
 	return nil
 }
 
+// WaitForProfileConditionStatus blocks until Profile with name `profile`
+// has applied TuneD profile `profileExpect` with its `conditionType` equal to
+// the value of 'conditionStatus'.
```

Comment:
I think this comment suggests that the function blocks until the TuneD profile is applied, but that isn't necessarily true. It blocks until the Profile object status condition `conditionType` matches  `conditionStatus` (regardless of whether the TuneD profile is applied or not, right? 

If you agree, maybe we can reword it to something like:
```suggestion
// WaitForProfileConditionStatus blocks until Profile with name `profile`
// is reporting its `conditionType` with the value of 'conditionStatus', 
// for the TuneD profile `profileExpect`.
```

Reply:
I agree, your comment is a lot more precise than the original.  Removed the trailing space and using it.  Thank you!